src/app/features/home/details/actions/actions.page.ts

```diff
@@ -142,9 +142,11 @@ export class ActionsPage {
         concatMap(orderId =>
           this.blockingActionService.run$(this.confirmOrder$(orderId))
         ),
-        tap(networkAppOrder =>
-          this.createOrderHistory$(networkAppOrder).subscribe()
-        ),
+        tap(networkAppOrder => {
+          if (Number(networkAppOrder.total_cost) !== 0) {
```

Comment:
Could you add comments to explain the workaround briefly or attach link to relevant issue in code? Might be easier for future readers to know why doing this here.